aws/table_aws_drs_source_server.go

```diff
@@ -198,3 +204,28 @@ func listAwsDRSSourceServers(ctx context.Context, d *plugin.QueryData, _ *plugin
 
 	return nil, nil
 }
+
+func getAwsDrsSourceServerLaunchConfiguration(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	sourceServer := h.Item.(types.SourceServer)
+
+	// Create Session
+	svc, err := DRSClient(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("aws_drs_source_server.getAwsDrsSourceServerLaunchConfiguration", "connection_error", err)
+		return nil, err
+	}
+
```

Comment:
```
	if svc == nil {
		// Unsupported region, return no data
		return nil, nil
	}
```
I think we need to add the service level check to make sure that the unsupported regions are handled correctly.

Reply:
No, we don't need that check except the list/get hydrated call. Other hydrated function will not get called until the list/get hydrated call get called.

Reply:
@ParthaI it is needed if the hydrate call is not supported in a particular region.